config/local-certs/rootCA.csr.cnf

```diff
@@ -0,0 +1,12 @@
+[req]
+distinguished_name = req_distinguished_name
+prompt = no
+
+[ req_distinguished_name ]
+countryName = US
+stateOrProvinceName = District of Columbia
```

Comment:
This is a file I made up, the other one comes from the old README instructions, do we think it's better to have "RandomCity" etc etc? I don't have strong feelings

Reply:
@jmhooper can you clarify the 👍 ? Yes keep "District of Columbia" or yes keep "RandomCity"?

Reply:
Misread this, I don't have strong feeling either :+1: is for "District of Columbia"